masterlist.yaml

```diff
@@ -6920,7 +6920,14 @@ plugins:
         condition: 'checksum("Hunterborn.esp", 7117CBA9)'
       - name: -ObjectBounds
         condition: 'checksum("Hunterborn.esp", 7117CBA9)'
+    dirty:
+      - <<: *quickClean
+        crc: 0xF1F2C258
+        util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 8
     clean:
+      - crc: 0xBED2E14D
+        util: 'SSEEdit v4.0.3'
```

Comment:
I don't usually add clean entries produced from dirty plugins.
They're volatile and may be of no use if xEdit is updated.

Reply:
Got it, I'll keep that in mind going forward.